storage/cache/subtree_cache.go

```diff
@@ -155,15 +155,14 @@ func (s *SubtreeCache) preload(ids []storage.NodeID, getSubtrees GetSubtreesFunc
 	want := make(map[string]storage.NodeID)
 	for _, id := range ids {
 		sInfo := s.stratumInfoForNodeID(id)
-		pxKey := id.PrefixAsKey(sInfo.prefixBytes)
+		pxID := id.Prefix(sInfo.prefixBytes)
+		pxKey := string(pxID.Path)
 		if _, ok := want[pxKey]; ok {
 			// No need to check s.subtrees map twice.
 			continue
 		}
 		if _, ok := s.subtrees.Load(pxKey); !ok {
-			// TODO(al): Fix for non-uniform strata.
```

Comment:
Before I review this: If "non-uniform" here means "not exactly 8 bits" then this could possibly break the map. If it means "not a multiple of 8 bits" then that has never worked and this is ok.

Reply:
I think it meant non-multiples-of-8.

Reply:
I thought we might have lost the quantum bit but it's reflected in the `sInfo` prefix.

I don't think "non uniform" was the right term in that comment but as it's gone it's no longer an issue.